src/modules/firewall/src/lib/Firewall.cpp

```diff
@@ -15,8 +15,46 @@ const std::string FirewallModuleBase::m_moduleInfo = R"""({
     "UserAccount": 0})""";
 
 const std::string FirewallModuleBase::m_firewallComponent = "Firewall";
+
 const std::string FirewallModuleBase::m_firewallReportedFingerprint = "firewallFingerprint";
 const std::string FirewallModuleBase::m_firewallReportedState = "firewallState";
+const std::string FirewallModuleBase::m_firewallReportedDefaultPolicies = "defaultPolicies";
+const std::string FirewallModuleBase::m_firewallReportedConfigurationStatus = "configurationStatus";
+const std::string FirewallModuleBase::m_firewallReportedConfigurationStatusDetail = "configurationStatusDetail";
+
+const std::string FirewallModuleBase::m_firewallDesiredDefaultPolicies = "firewallDesiredDefaultPolicies";
+const std::string FirewallModuleBase::m_firewallDesiredRules = "desiredFirewallRules";
+
+const char g_desiredState[] = "desiredState";
+const char g_action[] = "action";
+const char g_direction[] = "direction";
+const char g_protocol[] = "protocol";
+const char g_source[] = "sourceAddress";
+const char g_destination[] = "destinationAddress";
+const char g_sourcePort[] = "sourcePort";
+const char g_destinationPort[] = "destinationPort";
+
+const char g_desiredStatePresent[] = "present";
+const char g_desiredStateAbsent[] = "notPresent";
+
+const char g_actionAccept[] = "accept";
+const char g_actionReject[] = "reject";
+const char g_actionDrop[] = "drop";
+
+const char g_directionIn[] = "in";
+const char g_directionOut[] = "out";
+
+const char g_protocolAny[] = "any";
+const char g_protocolTCP[] = "tcp";
+const char g_protocolUDP[] = "udp";
+const char g_protocolICMP[] = "icmp";
+
+const char g_targetAccept[] = "ACCEPT";
+const char g_targetReject[] = "REJECT";
+const char g_targetDrop[] = "DROP";
+
+const char g_chainInput[] = "INPUT";
+const char g_chainOutput[] = "OUTPUT";
```

Comment:
Can do const char* g_chain[] = {"INPUT", "OUTPUT"}; or similar. Idea being: arrays of strings simulating enumerations, rather than standalone constants.